SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -365,16 +366,16 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	UFUNCTION(BlueprintPure, Category = "Spatial Functional Test")
 	bool WasLoadedFromTakenSnapshot();
 
-	template <typename T>
-	static int GetNumberOfActorsOfType(UWorld* World)
+	template <typename Actor_Type>
```

Comment:
Nit, could also be `TActorType`
```suggestion
	template <typename ActorType>
```

Reply:
+1